back/app/models/notifications/project_phase_started.rb

```diff
@@ -65,7 +65,7 @@ def self.make_notifications_on(activity)
       phase = activity.item
 
       if phase.project
-        user_scope = User.where.not(id: UserRoleService.new.moderators_for(phase))
+        user_scope = ParticipantsService.new.projects_participants(Project.where(id: phase.project_id))
         ActiveRecord::Base.transaction do
```

Comment:
This transaction (and the comment) is really confusing. We don't create anything here since this commit https://github.com/CitizenLabDotCo/citizenlab/commit/39508eb, so we don't need the transaction.

If you have time/eagerness, could you remove it?

Reply:
It was only added 2 weeks ago, by Sebi, so I suspect it is needed.

Reply:
I believe it was added three years ago https://github.com/CitizenLabDotCo/citizenlab/commit/f47e37c

Reply:
Sorry, I meant 'touched by' Sebi 2 weeks ago.

Reply:
I can remove it, if not needed, for sure.